pandas/tests/io/formats/test_to_html.py

```diff
@@ -99,6 +100,14 @@ def test_to_html_unicode(df, expected, datapath):
     assert result == expected
 
 
+def test_to_html_encoding():
+    df = DataFrame({"A": ["a", "b"]})
+    with tm.ensure_clean("test.csv") as path:
+        df.to_html(path, encoding="gbk")
+        with codecs.open(path, "r", encoding="gbk") as f:
```

Comment:
Is there a reason for using `codecs.open` here instead of just the built in `open`?